modules/setting/lfs.go

```diff
@@ -65,6 +67,14 @@ func newLFSService() {
 				cfg.Section("server").Key("LFS_JWT_SECRET").SetValue(LFS.JWTSecretBase64)
 			})
 		}
+
+		rootURL, parseErr := url.Parse(GetLFSRootURL())
+		if parseErr != nil {
+			log.Fatal("Failed to parse LFS root URL `%s`: %w", GetLFSRootURL(), parseErr)
+		} else if len(LFS.RootURL) > 0 {
+			rootURL.Path = "/"
```

Comment:
why do we have to set this?

Reply:
This ensures the trailing slash so that concatenations in `services/lfs/server.go` are guaranteed to do the right thing whether the user has a trailing slash in their config file or not.

Reply:
yes, would clarify things :)

Reply:
Done. Commit f927b00fa300d3251be03ace39eb198e38fe8a6e has some explanation as well.